client/scripts/views/modals/new_topic_modal.ts

```diff
@@ -67,6 +69,22 @@ const NewTopicModal: m.Component<{
               }
             }),
           ]),
+          m(FormGroup, [
```

Comment:
This shouldn't be visible for Substrate communities. Should check if Token Community